draft-ietf-quic-recovery.md

```diff
@@ -289,8 +289,14 @@ be subtracted from the RTT as long as the result is larger than the min_rtt.
 If the result is smaller than the min_rtt, the RTT should be used, but the
 ack delay field should be ignored.
 
-Like TCP, QUIC calculates both smoothed RTT and RTT variance similar to those
-specified in {{?RFC6298}}.
+A sender calculates both smoothed RTT and RTT variance similar to those
+specified in {{?RFC6298}}, see {{on-ack-received}}.
+
+A sender will commonly receive ACK frames that acknowledge larger packet numbers
+than before.  A sender might therefore take multiple samples per RTT (see
```

Comment:
Suggestion: replace "A sender..." on with: "When multiple samples are generated within an RTT, the smoothed RTT may retain inadequate RTT history, as suggested in {{?RFC6298}}.  Utilizing multiple RTT samples per RTT in smoothed RTT and RTT variance in a different way is currently an open research question."